owasp-top10-2021-apps/a9/games-irados/app/routes.py

```diff
@@ -120,12 +127,11 @@ def cupom():
         rows, success = database.get_game_coupon(coupon, session.get('username'))
         if not success or rows == None or rows == 0:
             flash("Cupom invalido", "danger")
+            app.logger.info("Invalid coupon: {}".format(coupon))
```

Comment:
Who is trying to use an invalid coupon? It would be nice to know for valid coupons as well

Reply:
fixed!